src/cdk/stack.ts

```diff
@@ -43,15 +43,15 @@ export class MiraServiceStack extends cdk.Stack {
     }
 
     /**
-   * Applies security policies.
-   */
+     * Applies security policies.
+     */
     applyPolicies (customList: any): void {
       this.node.applyAspect(new Policies(customList))
     }
 
     /**
-   * Initialize this component in some async way.
-   */
+     * Initialize this component in some async way.
```

Comment:
This doesn't inspire confidence! Either a better comment or a better function name is required.

Reply:
Yes, there are some comments present in the code that aren't exactly well suited for documentation 😅 . I will reword it.

Reply:
let's save this for later